test/functional/feature_pruning.py

```diff
@@ -124,7 +124,7 @@ def reorg_test(self):
         # Reboot node 1 to clear its mempool (hopefully make the invalidate faster)
         # Lower the block max size so we don't keep mining all our big mempool transactions (from disconnected blocks)
         self.stop_node(1)
-        self.start_node(1, extra_args=["-maxreceivebuffer=20000","-blockmaxsize=5000", "-checkblocks=5", "-disablesafemode"])
```

Comment:
Likewise, this should set blockmaxweight=20000 in keeping with the comment above about not mining big mempool transactions.